portfolio/src/main/java/com/google/sps/servlets/CreatePuzzleServlet.java

```diff
@@ -47,8 +48,6 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response)
       throws ServletException, IOException {
 
-    /* Upload the image to blob store */
-
     // Get the difficulty indicated by user.
     String difficulty = request.getParameter("difficulty");
```

Comment:
Don't forget to parse the string into an enum:
Difficulty difficulty = Difficulty.valueOf(request.getParameter("difficulty"));

Reply:
Also do note that valueOf() will throw an IllegalArgumentException if the string cannot be mapped to an enum. You can wrap it in a try catch and set an error message on the response then:
```java
  Difficulty difficulty;
  try {
      difficulty = Difficulty.valueOf(request.getParameter("difficulty"));
  catch (IllegalArgumentException e) {
     response.getWriter().println("Invalid value for difficulty");
     return;
  }
```